Dockerfile

```diff
@@ -4,6 +4,8 @@ WORKDIR /app
 
 RUN apt-get update && apt-get install -y default-libmysqlclient-dev
 
+COPY ./assets ./assets
+
```

Comment:
can we mount this in the docker-compose ymls like
```yml
volumes:
- ./assets:/app/assets
```

then we can remove this line